samples/accounts_user_links_batch_get_test.py

```diff
@@ -0,0 +1,28 @@
+# Copyright 2021 Google LLC All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import os
+
+import accounts_user_links_batch_get
+
+TEST_ACCOUNT_ID = os.getenv("GA_TEST_ACCOUNT_ID")
+TEST_USER_LINK_ID = os.getenv("GA_TEST_USER_LINK_ID")
+
+
+def test_accounts_user_links_batch_get(capsys):
+    accounts_user_links_batch_get.batch_get_account_user_link(
+        TEST_ACCOUNT_ID, TEST_USER_LINK_ID
+    )
+    out, _ = capsys.readouterr()
+    assert "Result" in out
```

Comment:
is the value you're expecting known? Could you add more to this assert if it's the same test value you're looking for every time? 

(Should you add a value in a fixture that you then "get" in the test and delete in the teardown?)